example_sqlite_test.go

```diff
@@ -36,9 +35,7 @@ func Example_sqlite() {
 	river.AddWorker(workers, &SortWorker{})
 
 	riverClient, err := river.NewClient(driver, &river.Config{
-		FetchCooldown:     20 * time.Millisecond,
-		FetchPollInterval: 50 * time.Millisecond, // this driver is poll only, so speed up poll interval so the test runs fater
```

Comment:
The most immediate advantage of this change for the test suite is that when testing SQLite, we don't make the tests super slow if we accidentally omit configuration for increased polling frequency. It can now be safely removed and we can be sure the producer will pick changes up right away.